Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/GroupListPickerViewController.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  GroupListPickerViewController.swift
+//  StreamChatUI
+//
+//  Created by Parth Kshatriya on 08/04/22.
+//
+
+import UIKit
+import StreamChat
+import Nuke
+
+class GroupListPickerViewController: UIViewController {
+
+    //MARK: Outlets
+    @IBOutlet weak var tblMember: UITableView! {
+        didSet {
+            tblMember.register(TableViewCellChatUser.nib, forCellReuseIdentifier: TableViewCellChatUser.reuseId)
+        }
+    }
+
+    //MARK: Variables
+    var controller: ChatChannelController?
+    var didSelectMember: ((ChatChannelMember) -> Void)?
+    private var members = [ChatChannelMember]()
+    private var currentPage : Int = 0
+    private var totalCount: Int = 0
+    private var isLoadingList : Bool = false
+    private var channelMember: ChatChannelMemberListController?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        guard let cid = controller?.cid else { return }
```

Comment:
@mdhanif-simformsolutions  Pls create setupUI() function and move these code there.

Reply:
Removed this controller as it is not required for now.